server/api/models/rpaTaskModel.js

```diff
@@ -11,9 +11,9 @@ const rpaParameterSchema = new Schema({
 });
 
 const rpaTaskSchema = new Schema({
-  Application: { type: String, required: [true, 'Application required'] },
-  Task: { type: String, required: [true, 'Task required'] },
-  Code: { type: String, required: [true, 'Code required'] },
+  application: { type: String, required: [true, 'Application required'] },
+  task: { type: String, required: [true, 'Task required'] },
+  code: { type: String, required: [true, 'Code required'] },
   outputValue: Boolean,
   inputVars: [rpaParameterSchema],
   Output: rpaParameterSchema,
```

Comment:
Output should also be written small: "output". Also change all occurences of this Output property in the code.